.github/workflows/tests_lkcpu_python.yml

```diff
@@ -249,8 +249,8 @@ jobs:
             PL_DEVICE=${DEVICENAME} python -m pytest tests/ $COVERAGE_FLAGS --splits 7 --group ${{ matrix.group }} \
               --store-durations --durations-path='.github/workflows/python_lightning_kokkos_test_durations.json' --splitting-algorithm=least_duration
             mv .github/workflows/python_lightning_kokkos_test_durations.json ${{ github.workspace }}/.test_durations-${{ matrix.exec_model }}-${{ matrix.group }}
-          pl-device-test --device ${DEVICENAME} --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
-          pl-device-test --device ${DEVICENAME} --shots=None --skip-ops $COVERAGE_FLAGS --cov-append
+          : # pl-device-test --device ${DEVICENAME} --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
+          : # pl-device-test --device ${DEVICENAME} --shots=None --skip-ops $COVERAGE_FLAGS --cov-append
```

Comment:
These lines are commented to not interrupt the CI workflows to check the sanity of the implementation. When the integration of new device API for LKokkos has been completed then these files will go to normality.

Reply:
I based on this [link](https://stackoverflow.com/questions/66150315/how-do-i-add-a-comment-in-github-action-yml-file). Also, I tried using just `#` and works well. However, @maliasadi suggests using it all the time.

Reply:
Thanks!